src/assignments/orbit-report.rst

```diff
@@ -1,282 +1,351 @@
+   following.
 
-.. sourcecode:: html+ng2
-   :linenos:
+   .. sourcecode:: html+ng2
+      :linenos:
 
-   <h3>Orbit Report</h3>
-   <table>
-      <tr class="header-row">
-         <th class="sortable">Name</th>
-         <th class="sortable">Type</th>
-         <th>Operational</th>
-         <th>Orbit Type</th>
-         <th>Launch Date</th>
-      </tr>
-      <!-- TODO: put <tr *ngFor=""></tr> here -->
-   </table>
+      <h3>Orbit Report</h3>
+      <table>
+         <tr class="header-row">
+            <th class="sortable">Name</th>
+            <th class="sortable">Type</th>
+            <th>Operational</th>
+            <th>Orbit Type</th>
+            <th>Launch Date</th>
+         </tr>
+         <!-- TODO: put <tr *ngFor=""></tr> here -->
+      </table>
 
-Next you need to include CSS that will make your table and application look a little nicer.
+Next, you need to include CSS that will make your table and application look a
+little nicer.
 
-2. Copy the entire contents of the `example orbit-list-component.css  <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`_ and put it into your ``orbit-list-component.css``.
-3. Copy the entire contents of the `example app.component.css <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`_ and put it into your ``app.component.css``.
-4. Copy the ``<style>`` tag from the `example index.html <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`_ and paste it into the ``<head>`` of your ``index.html``.
+2. Copy the entire contents of the example `orbit-list-component.css  <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`__
+   and put it into your ``orbit-list-component.css``. Leave the ``sortable``
+   class alone, but change the ``table`` and ``warning`` styles to colors
+   and sizes you find pleasing.
+#. Copy the entire contents of the example `app.component.css <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`_ and put it into your ``app.component.css``.
+#. Copy the ``<style>`` tag from the example `index.html <https://gist.github.com/welzie/5247f5ac36e973903cd5202af50932e6>`_ and paste it into the ``<head>`` of your ``index.html``.
 
-   * Only add in the ``<style>``, do NOT remove any other HTML from your ``index.html``.
+   a. Only add in the ``<style>``, do NOT remove any other HTML from your
+      ``index.html``.
 
-5. View the app in your browser. You should see a table like the below.
+#. View the app in your browser. Your table should look something like:
 
-.. figure:: figures/basic-table-satellites.png
-   :alt: Screen shot of browser showing http://localhost:4200 with a table of four satellites.
-
-   Example of application at this point. Don't worry if yours doesn't have a heading of "Orbit Report" yet, you can add that later.
+   .. figure:: figures/basic-table-satellites.png
+      :alt: Screen shot of browser showing http://localhost:4200 with a table of four satellites.
```

Comment:
Hmmm... So the data is shifted over by one cell?  I don't think this is a Chrome vs. Firefox/Safari thing, but can you confirm if the shift appears when you view the project in a different browser?

I demoted my project back to the hard-coded satellite array, and I am not seeing the column shift in Firefox or Safari.  Here is my code for ``orbit-list.component.html``:

`<h3>Orbit Report</h3>
<table>
   <tr class="header-row">
      <th class="sortable">Name</th>
      <th class="sortable">Type</th>
      <th>Operational</th>
      <th>Orbit Type</th>
      <th>Launch Date</th>
   </tr>
   <tr *ngFor="let satellite of satellites">
      <td>{{satellite.name}}</td>
      <td>{{satellite.type}}</td>
      <td>{{satellite.operational}}</td>
      <td>{{satellite.orbitType}}</td>
      <td>{{satellite.launchDate}}</td>
  </tr>
</table>`

Reply:
yeah the column shift doesnt happen in firefox. and it will resolve itself in chrome once a search is made, which is also weird.
i dont think its critical enough to prevent this from being merged since we tell the students to use firefox for the class

Reply:
OK.  We will have to keep our eyes on this one.